src/service/ReservationService.java

```diff
@@ -0,0 +1,101 @@
+package service;
+
+import model.Customer;
+import model.IRoom;
+import model.Reservation;
+
+import java.util.*;
+
+public class ReservationService {
+    public static final ReservationService reservationService = new ReservationService();
```

Comment:
It's usually not great if something has a static instance of itself (in which case a class manages its own lifecycle). 
At least, make it private and make it accessible via a static `getInstance()` method.

If it is required for this exercise, keep this singleton pattern, but make sure you read on articles the downsides of the Singleton pattern. For example: https://refactoring.guru/design-patterns/singleton